assets/components/CalendarButton.tsx

```diff
@@ -24,7 +26,21 @@ class CalendarButton extends React.Component<any, any> {
         prevProps.activeDate === EARLIEST_DATE && this.setState({startDate: moment(this.props.activeDate)});
     }
 
+    getLocale = () => {
+        const locale = window.locale.replace('_', '');
+        const rootLocale = locale.substring(0, 2);
+        const allLocales = locales as any;
+
+        return allLocales[locale] || allLocales[rootLocale];
```

Comment:
You can use `??` if `allLocales[rootLocale]` is intended to be a fallback, `||` is a very old way of doing it